tests/functional/deprecations/01-cylc8-basic/validation.stderr

```diff
@@ -48,5 +48,5 @@ WARNING -  * (8.0.0) [cylc][events]abort if inactivity handler fails - DELETED (
 WARNING -  * (8.0.0) [cylc][events]abort if stalled handler fails - DELETED (OBSOLETE)
 WARNING -  * (8.0.0) [cylc] -> [scheduler] - value unchanged
 WARNING - deprecated graph items were automatically upgraded in "workflow definition":
-     * (8.0.0) [scheduling][dependencies][X]graph -> [scheduling][graph]X - for X in:
+     * (8.0.0) [scheduling][dependencies][X]graph -> [scheduling][graph] - for X in:
            P1D
```

Comment:
This doesn't seem right. The deprecation warning was correct before

Reply:
agreed - fixing